cmd/wallet-web/src/main.js

```diff
@@ -33,6 +33,7 @@ Vue.use(SideBar);
 Vue.use(VueCookies);
 Vue.$cookies.config('7d');
 Vue.use(VueMaterial);
+Vue.component('ToastAlert', require('./components/ToastNotification/ToastNotification').default);
```

Comment:
Since the component is called `ToastNotification`, shouldn't we register it under the same name here to be consistent? (Instead of the `ToastAlert`)